worktree.go

```diff
@@ -262,6 +263,31 @@ func (w *Worktree) setHEADToBranch(branch plumbing.ReferenceName, commit plumbin
 	return w.r.Storer.SetReference(head)
 }
 
+// unstages a set of files -- equivalent to "git restore --staged <file>..."
+func (w *Worktree) Restore(o *RestoreOptions) error {
```

Comment:
Please document this method following the documentation standards

Reply:
Commit eac1e381cdd4769303231f06218a7eb1610f8b76